README.md

```diff
@@ -100,6 +100,37 @@ Ensure you have "killall" installed on your system (*psmisc* package on Debian-l
 
 If the node and Gateway are on different systems, you may need to utilize the "NOTIFY" endpoint on the Gateway's dashboard/API instead.
 
+## Docker
+
+Clone and build image, Create your config file, see the config_template.json. then:
+
+  ```
+  docker run -d \
+  -v ./YOUR_CONFIG.json:/app/datum_gateway_config.json:ro \
+  -p 8080:8080 -p 23334:23334 \
+   datumgateway
+  ```
+
+OR Compose Service:
+
+  ```
+  version: '3'
+  services:
+    datumgateway:
+      build: 
+        context: https://github.com/OCEAN-xyz/datum_gateway.git
```

Comment:
Does this clone from git blindly? What would stop GitHub, if they were malicious, simply replacing the code without any further user interaction?

Reply:
But if the user has this, they already did a git checkout, right? Can we tell it to just use the branch checked out already?

Reply:
But if the user has this, they already did a git checkout, right? Can we tell it to just use the branch checked out already?